src/main/java/ca/spottedleaf/starlight/mixin/common/lightengine/LevelLightEngineMixin.java

```diff
@@ -178,28 +178,24 @@ public void clientUpdateLight(final LightLayer lightType, final SectionPos pos,
         // data storage changed with new light impl
         final ChunkAccess chunk = this.getLightEngine().getAnyChunkNow(pos.getX(), pos.getZ());
         switch (lightType) {
-            case BLOCK: {
-                final SWMRNibbleArray[] blockNibbles = this.blockLightMap.computeIfAbsent(CoordinateUtils.getChunkKey(pos), (final long keyInMap) -> {
-                    return StarLightEngine.getFilledEmptyLight(this.lightEngine.getWorld());
-                });
+            case BLOCK -> {
```

Comment:
Again, too long

Reply:
This is a Java 14 enhanced Switch its supposed to make it look nicer

Reply:
but I can revert that if you guys want to

Reply:
I meant to click the line below - my bad 😣

Reply:
all good lol